src/sys.h

```diff
@@ -7,6 +7,31 @@ extern "C" {
 
 extern long get_available_space(const char *path);
 
+/* run_command: runs a command redirecting the standard and error outputs
```

Comment:
Minor nit, but this comment probably means to say "run_command_quiet: runs a command...."